daisy/prow_images/functional_tests/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:stretch
```

Comment:
Maybe see of you can use a smaller image, maybe 'python-2.7-alpine', you'll need to update the package installs but it should have everything.